docs/articles/documentation/using-testcafe/common-concepts/concurrent-test-execution.md

```diff
@@ -21,6 +21,8 @@ These instances constitute the pool of browsers against which tests run concurre
 To enable concurrency, use the [-c or --concurrency](../command-line-interface.md#-c-n---concurrency-n)
 command line option or the [runner.concurrency](../programming-interface/runner.md#concurrency) method of the programming interface.
 
+> Important! Concurrent test execution is not supported in Microsoft Edge.
```

Comment:
Maybe we should describe why?

Reply:
If we do, and when MS allows invoking multiple Edge windows, this note will immediately become false.

If we don't, it will still be true until we change `browser-natives` to support MS Edge.

That's why I don't like documenting the functionality of someone else's products.